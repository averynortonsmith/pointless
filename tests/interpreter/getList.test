
output = [a, b, c, d, e] |> println

a = [1, 2, 3].!getList
b = {1, 2, 3}.!getList
c = [1 2 3].!getList
d = "asdf".!getList
e = (1, 2, 3).!getList