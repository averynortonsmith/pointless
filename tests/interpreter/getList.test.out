[[1, 2, 3], [1, 2, 3], [1, 2, 3], ["a", "s", "d", "f"], [1, 2, 3]]
